oak_sev_guest/src/guest.rs

```diff
@@ -71,7 +71,7 @@ impl GuestMessage {
 
 /// The authenticated subsection of the header used for an encrypted guest request message.
 ///
-/// See Table 97 in <https://www.amd.com/system/files/TechDocs/56860.pdf>.
+/// See Table 99 in <https://www.amd.com/system/files/TechDocs/56860.pdf>.
```

Comment:
out of curiosity, did the table numbers change since last time, or did we write them down incorrectly? Just to have an idea of how often that PDF changes

Reply:
If only we had a way of referring to stuff on the internet by digest ... :)